src/app.module.ts

```diff
@@ -3,24 +3,26 @@ import { ConfigModule, ConfigService } from '@nestjs/config';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
-import { configConstant } from './common/constants/config.constant';
+import { ApplicantsModule } from './applicants/applicant.module';
 
 @Module({
   imports: [
+    ApplicantsModule,
     ConfigModule.forRoot({ isGlobal: true }),
     TypeOrmModule.forRootAsync({
       imports: [ConfigModule],
       useFactory: (configService: ConfigService) => ({
         type: "postgres",
-        host: configService.get(configConstant.database.host),
-        port: configService.get(configConstant.database.port),
-        username: configService.get(configConstant.database.username),
+        host: configService.get<string>("POSTGRES_HOST"),
```

Comment:
why did u change this from using the values in the config constant

Reply:
Error: SASL: SCRAM-SERVER-FIRST-MESSAGE: client password must be a string

this is the error i get when i use the values from the config constants

whereas when i loaded the values from a separate .env file it worked fine

Reply:
That is probably because you're doing something wrong. Th values in the config constant must those in the .env file case wise